faustjs/tutorial/setup-faustjs.mdx

```diff
@@ -20,7 +20,7 @@ You can download and install WPGraphQL from the [WordPress Plugin Directory](htt
 
 The WPE Headless plugin ensures that your WordPress site runs smoothly as a headless CMS. This includes things like smart content redirects and enabling post previews.
 
-To install WPE Headless, [download the latest version](https://wp-product-info.wpesvc.net/v1/plugins/wpe-headless?download) and activate the plugin through the WordPress admin.
+To install WPE Headless, [download the latest version](https://wp-product-info.wpesvc.net/v1/plugins/wpe-headless?download). Then, upload and activate the plugin from within the WordPress admin.
```

Comment:
This one is more of a nit, but I think we should refer to the WordPress UI as the "WordPress admin panel". WordPress admin implies a person or role, instead of a UI.